element_session/session_with_id.py

```diff
@@ -81,3 +82,17 @@ class ProjectSession(dj.Manual):
     -> Project
     -> Session
     """
+
+@schema
+class Location(dj.Manual):
+    definition = """
+    location: varchar(32)
+    ---
+    location_full='': varchar(1000)
+    """
+  
+@schema
+class Equipment(dj.Manual):
+    definition = """
+    scanner: varchar(32)
+    """

```

Comment:
The `adamacs/schemas` already define `Location` and `Equipment` when importing to the [pipeline namespace here](https://github.com/SFB1089/adamacs/blob/main/adamacs/pipeline.py#L7-L8). I would avoid multiple tables with the same name. Is there a reason to preserve them here with session vs. under adamacs schemas? I would default to the latter